schemas/telemetry/debug-schema.json

```diff
@@ -13,9 +13,14 @@
       "properties": {
         "telemetry": {
           "type": "object",
-          "description": "The telemetry information",
+          "description": "The telemetry log information",
```

Comment:
Why are you changing some telemetry related schema here?

Reply:
oh...not sure how this happened....I am going to check on my end.